src/components/RightSidebar/RightSidebar.astro

```diff
@@ -14,19 +14,14 @@ const headers = content.astro?.headers;
 <nav class="sidebar-nav" aria-label={t('rightSidebar.a11yTitle')}>
 	<div class="sidebar-nav-inner">
 		{headers && (
-			<TableOfContents
-				client:media="(min-width: 50em)"
-				headers={headers}
-				labels={{ onThisPage: t('rightSidebar.onThisPage'), overview: t('rightSidebar.overview') }}
-			/>
+		<TableOfContents client:media="(min-width: 50em)" headers={headers} labels={{ onThisPage:
+			t('rightSidebar.onThisPage'), overview: t('rightSidebar.overview') }} />
 		)}
 		<ContributeMenu editHref={githubEditUrl} />
-		<CommunityMenu/>
+		<CommunityMenu />
 		<div style="margin: 2rem 0; text-align: center;">
-			<ThemeToggleButton 
-				client:visible 
-				labels={{  useLight: t('themeToggle.useLight'), useDark: t('themeToggle.useDark')  }} 
-			/>
+			<ThemeToggleButton client:visible labels={{ useLight: t('themeToggle.useLight'), useDark:
+				t('themeToggle.useDark') }} />
```

Comment:
Ooh, this formatting is a little odd. Is this the auto-formatting the VS Code extension now provides?

Reply:
Maybe it's from Gitpod? I will fix this real quick.